plugins/modules/aws_kms_info.py

```diff
@@ -93,7 +93,10 @@
       returned: always
       sample: arn:aws:kms:ap-southeast-2:123456789012:key/abcd1234-abcd-1234-5678-ef1234567890
     key_state:
-      description: The state of the key
+      description:
+        - The state of the key
+        - Will be one of C('Creating'), C('Enabled'), C('Disabled'), C('PendingDeletion'), C('PendingImport'),
+          C('PendingReplicaDeletion'), C('Unavailable'), or C('Updating')
```

Comment:
You can also use `choices`.
```suggestion
          C('PendingReplicaDeletion'), C('Unavailable'), or C('Updating').
```

Reply:
@alinabuzachis can you use choices for a return val?

Reply:
Oh, I realised now it is a returned val. I don't think so.